### ISSUE TITLE
Discourse Connectives

### ISSUE THREAD

**Issue by timjogorman** [amrisi/amr-guidelines#112]
Hii all!  I wanted to put up a list of proposed discourse connective changes.  A list of a few common connectives, with proposed senses ranked by rough frequency, is at bottom.
- **Directed alternatives:**
  I'd propose using "replace-01" and ":alternative"  for connectives like "instead of", "rather than", "in place of", "in lieu of", etc.    Most of these look to now be :prep-instead-of in the release set..
- **Exceptions:**
  I'd propose "exclude-01 /  :exception" and used for connectives like except, besides, excluding, but for, otherwise, etc, as in

> I've seen otherwise intelligent people convinced that ....
> 
> ``` lisp
>     (s / see-01
>             :ARG0 (i / i)
>             :ARG1 (c / convince-01
>                   :ARG0 (p / person
>                         :ARG1-of (i2 / intelligent-41
>                               :exception c
> ```
- **Disjunction:**
  Splitting between inclusive 'or' and exclusive 'or' seems like a gargantuan task, and perhaps not always clear.  However, in figuring that out, I've noticed that we tend to use an "if not X, Y" pattern for the disjunctions where that makes sense.  I'd propose having some frames (particularly for "otherwise") have  that suggestion -- use of have-condition-91 along with negation on the condition -- as an alternative. This is one of those things we often seem to already do:

> Health and safety and environmental horrors need to be regulated otherwise industries will not EVER do whats right for humanity.
> 
> ``` lisp
> (h / have-condition-91
>       :ARG1 (d / do-02 :polarity -
>             :ARG0 (i / industry)
>             :ARG1 (t / thing
>                   :ARG1-of (r / right-42
>                         :ARG2 (h2 / humanity)))
>             :time (e / ever))
>       :ARG2 (n / need-01
>             :ARG0 (a / and
>                   :op1 (h3 / horror
>                         :mod (h4 / health))
>                   :op2 (h5 / horror
>                         :mod (s / safe-41))
>                   :op3 (h6 / horror
>                         :mod (e2 / environment)))
>             :ARG1 (r2 / regulate-01
>                   :ARG1 a)))
> ```
- I'd propose  we adopt "before" as a consistent way of encoding things being **in temporal sequence** -- perhaps using an actual predicate like in-temporal-sequence-91 or predate-01.  Part of this is the mere convenience that if  you present a list of options for things like "once" or "since", one of those options should be the temporal expression.
- I'd propose using  "continue-sequence-91" or ":sequential" for anything marking something **in sequence** that doesn't fit more meaningful categories, for things like furthermore, moreover, besides, etc.
- Finally, I'd propose that we expand exemplify-01 to all **instance+generalization** relationships.  This is most tricky with things like

> Since 9 / 11 , Middle Easterners in general , and Iraqis in particular , have faced enormous hurdles getting admitted to the United States ;
> 
> ``` lisp
> (f / face-01
>   :ARG0 (a / and
>           :op1 (p / person
>                  :mod (w / world-region
>                         :name (n3 / name
>                                 :op1 "Middle"
>                                 :op2 "East"))
>                  :prep-in (g / general))
>           :op2 (p2 / person
>                  :mod (c2 / country
>                         :name (n4 / name
>                                 :op1 "Iraq"))
>                  :prep-in (p3 / particular)))
>   :ARG1 (h / hurdle
>           :mod (e2 / enormous)
>           :time (g2 / get-03
>                   :ARG1 (a2 / admit-02
>                           :ARG1 a
>                           :ARG2 (c3 / country
>                                   :name (n6 / name
>                                           :op1 "United"
>                                           :op2 "States")))))
>   :time (s / since
>           :op1 (e3 / event
>                  :name (n5 / name
>                          :op1 "9/11"))))
> ```
### Examples of sense suggestions for connectives

while: be-temporally-at.91, contrast.01, have-concession.91 
however: contrast.01, have-concession.91
with respect to: concern-02
instead: replace.01, contrast.01
once: in-temporal-sequence-91, have-condition.91
since: cause.01, have-purpose-91, in-temporal-sequence-91
but: contrast.01, have-concession.91, exclude.01
otherwise: exclude.01, { have-conditional-91 :arg2 have-polarity-91 }
in other words: mean.01, exemplify.01 
in short: exemplify.01

**Comment by nschneid** [comment]
I was looking at the treatment of __still__ in the data, and rediscovered this issue. 

The treatment is not consistent, and I agree something more semantic is desirable. Right now it is often `:mod`, but sometimes `:time` and even (in 2 release AMRs) the ARG2 of `contrast-01`!

@timjogorman proposes 3 senses, mapped to `have-concession-91`, `:time`, and `contrast-01`. I'm not sure I see the difference between the first and third of these.

There are cases which I think are primarily temporal, but further suggest a contrast ("I stop the multivitamin and my mouth symptoms are gone, sadly my eyesight is still damaged."). There are also other cases which are clearly not temporal—e.g., the headline [Yes, I’m a nine-year-old girl. But I’m still a serious reporter](http://www.theguardian.com/commentisfree/2016/apr/06/nine-year-old-reporter-orange-street-news-truth).

Could we test for `:time` with an "it remains the case that..." paraphrase?

I think there are also cases that are similar to the temporal sense, but not quite temporal. Imagine the discourse: "Landlord: I can give you a discounted rate of $2000/mo. Prospective tenant: No, $2000/mo is still too expensive for me." Maybe this is discourse-time?

**Comment by timjogorman** [comment]
I'd be fine with any treatment of "still" that gets us out of doing ":mod (s / still)"!  My assumption of splitting "still" into concession and contrast-01 was purely data-driven; PDTB annotations seem to split it roughly down the middle.  

As for the gradient of differences between temporal and concessive still: I'd love for us have (one day) a rich treatment of these "phasal particle" or "temporal expectation" operators, like *still*, *already*, *yet*/*not yet*, *no longer*, *anymore* (and *deja*, *schon*, *noch*, etc). But unless someone has some brilliant proposal for what that would be, a test like "it remains the case that..." sounds like the only easy boundary to draw. 

> "Or something about scales?"

See [Michaelis '93](http://spot.colorado.edu/~michaeli/documents/Michaelis_still.pdf) for a scalar-model  treatment of still :).  It has some "scalar" examples that seem similar to that:
- "I can still beat Paul. Peter is too good for me."
- "Compacts are still pretty safe; subcompacts start to get dangerous"
